apps/heft/src/plugins/Webpack/WebpackPlugin.ts

```diff
@@ -70,6 +69,13 @@ export class WebpackPlugin implements IHeftPlugin {
         port: 8080
       };
 
+      if (Array.isArray(webpackConfiguration)) {
+        // TODO: How should we handle the MultiCompiler. Do we need to spawn a
```

Comment:
@iclanton - Do you have any guidance as to how we should handle the multiple configs in the context of a dev server?

Reply:
What does WDS do? I can look into this a little more tomorrow.

Reply:
Maybe we take the first one, log that we're doing that, and emit a warning if more than one is specified?

Reply:
yup, that works!